quickstart/overview.markdown

```diff
@@ -3,24 +3,24 @@ title: Overview of the ALICE O<sup>2</sup>
+This is an overview of the O<sup>2</sup> system. It is pretty rough, but it should give you an idea of the general architecture. Please refer to other documents such as the [TDR](https://cds.cern.ch/record/2011297).
 
 ## Requirements
 
-1. After LS2 the ALICE experiment will have gone through a massive upgrade to handle minimum bias PbPb interactions at 50 kHz what results in a 100 times more data than in 2010.
+1. After LS2 the ALICE experiment will have gone through a massive upgrade to handle minimum bias PbPb interactions at 50 kHz. This will results in a 100 times more data than in Run 2.
 
-2. Moreover, the physics topics addressed by the ALICE upgrade are characterised by a very small signal-to-noise ratio, requiring very large statistics, and a large background, which makes triggering techniques very inefficient, if not impossible.
+2. The physics topics addressed by the ALICE upgrade are characterised by a very small signal-to-noise ratio, requiring very large statistics, and a large background, which makes triggering techniques very inefficient, if not impossible.
 
-3. Finally, the TPC has an inherent rate <50Hz requiring a continuous, _non-triggered_, read-out.
+3. Finally, the TPC's inherent rate of ~50Hz requires a continuous, _non-triggered_, read-out.
 
 ## A new computing system : ALICE O<sup>2</sup>
 
- For these 3 main reasons, the amount of data to be handled is so large (3.4TB/s) that it is not possible to simply store it as we did during Run 1 and 2.
+ Due to above requirements the readout throughput increased to 3.4TB/s. Such amount of data is difficult to store as it was done during Run 1 and 2.
 
- This lead us to design a new computing system for Run 3 and 4. In this new system, all interactions are read-out. To handle the large amount of data, 3.4 TB/s, and store it, we need to compress it intelligently by reconstructing the data online. By doing so, we achieve more than a factor 30 compression and the traditional boundary between online and offline system blurs out. As a consequence, we have a common computing system for online and offline, thus the name “O2”.  
+ This lead us to design of a new computing system for Run 3 and 4. In this very new system we compress the data on-the-fly by partial online reconstruction achieving a compression factor of 30. The unified design causes that traditional boundary between online and offline system blurs out. As a consequence, we have a common computing system for online and offline, thus the name “O2”.  
 
-## Data Flow
+## Data flow
 
-The data flow is depicted below in a very simplified way. The first row of machines is made of __FLPs__ and the second row of __EPNs__. A much more detailed description and figure is available in the [TDR](https://cds.cern.ch/record/2011297).
+The data flow is depicted below in a simplified way. A much more detailed description and figure is available in the [TDR](https://cds.cern.ch/record/2011297).
```

Comment:
Ok, then I will update the picture to show "FLPs" and "EPNs".

Reply:
Yup I deleted it as you can't see neither FLP nor EPN in the figure.